pom.xml

```diff
@@ -167,6 +167,11 @@
         <version>1.48</version>
       </dependency>
       <dependency>
+        <groupId>com.fasterxml</groupId>
+        <artifactId>classmate</artifactId>
```

Comment:
It's part of the transitive dependencies of using jersey's jersey-bean-validation and hibernate-validator default implementation (along with the javax.el deps, jboss-logging dep, etc.)